WinCompatibilityPack/WinCompatibilityPack.csproj

```diff
@@ -1,6 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFrameworks>netcoreapp20;net472</TargetFrameworks>
```

Comment:
since we now manually copy the assemblies, lines 12 and 13 can be removed as those will need to be manually copied in the build script as well.